src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -515,67 +515,79 @@ Iteration <number>:
               -DCMAKE_BUILD_TYPE=Release                    \
               -DWITH_OPENVINO=ON                            \
-              -DOpenVINO_DIR=<path-to-openvino-install-dir> \
               -DWITH_ONNX=ON                                \
               -DORT_INSTALL_DIR=<path-to-onnxrt-install-dir>
     cmake --build . --config Release --target opencv_gapi --parallel
-	```
-### In-plugin build
-
-1. Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
-2. Build OpenCV G-API with OpenVINO / ONNXRT support
-3. Clone `npu-plugin` repository
-	```
-	git clone https://github.com/openvinotoolkit/npu_plugin
-	git submodule update --init --recursive
-	```
-4. Build `Protopipe` as part of the `npu-plugin` build:
-	```
-	mkdir build && cd build
-	cmake ../ -DOpenCV_DIR=<path-to-opencv-build> -DOpenVINODeveloperPackage_DIR=<path-to-openvino-build>
-	cmake --build . --config Release --target protopipe --parallel
-	```
-
+    ```
+#### Build Protopipe inside openVino
+1. Build protopipe
+    ```
+    cd <path-to-OpenVINO-root-dir>
+    mkdir "src/plugins/intel_npu/tools/protopipe/build" && cd "src/plugins/intel_npu/tools/protopipe/build"
+    cmake ../ -DOpenCV_DIR=<path-to-opencv-build-dir> -DCMAKE_BUILD_TYPE=Release
+    cmake --build . --target protopipe --parallel --config Release
+    ```
 ### Standalone build
 1. Build `yaml-cpp`
-	```
-	mkdir -p yaml-cpp_build cd && yaml-cpp_build
-	cmake ../<npu-plugin>/thirdparty/yaml-cpp -DCMAKE_INSTALL_PREFIX=install
-	cmake --build . --config Release --target install --parallel
-	```
+   ```
+   mkdir -p "yaml-cpp_build" && cd "yaml-cpp_build"
+   cmake <openvino-dir>/src/plugins/intel_npu/thirdparty/yaml-cpp -DCMAKE_INSTALL_PREFIX=install
+   cmake --build . --config Release --target install --parallel
+   ```
 2. Build `gflags`
-	```
-	git clone https://github.com/gflags/gflags
-	cd gflags
-	mkdir -p gflags_build cd && gflags_build
-	cmake ../ -DCMAKE_INSTALL_PREFIX=install
-	cmake --build . --config Release --target install --parallel
-	```
+   ```
+   git clone https://github.com/gflags/gflags
+   cd gflags
+   mkdir -p "gflags_build" && cd "gflags_build"
+   cmake ../ -DCMAKE_INSTALL_PREFIX=install
+   cmake --build . --config Release --target install --parallel
+   ```
 3. Build `Protopipe`
-	```
-	mkdir -b protopipe_build && cd protopipe_build
-	cmake <npu-plugin>/tools/protopipe/                              \
-	      -DOpenCV_DIR=<path-to-opencv-build                         \
-	      -Dyaml_cpp_DIR=<yaml-cpp_build/install/lib/cmake/yaml-cpp> \
-	      -Dgflags_DIR=<gflags_build/install>                        \
-	      -DOpenVINO_DIR=<path>                                      \
-	           
-	cmake --build . --config Release --target protopipe --parallel
-	```
+   ```
+   mkdir -p "protopipe_build" && cd "protopipe_build"
+   cmake <openvino-dir>/src/plugins/intel_npu/tools/protopipe \
+         -DOpenCV_DIR=<path-to-opencv-build>                  \
```

Comment:
might be `<opencv-build-dir>` could work here also (as in lines below)

Reply:
The folder could be anywhere that is true. As I noticed in all Intel project the unwritten rule is to put the build folder in the route folder of the git source. Now protopipe is treated as a subproject so the build folder should be relative to the source folder.